static/scss/post-list.scss

```diff
@@ -13,3 +13,55 @@
     color: $font-grey;
   }
 }
+
+.post-toprow {
+  min-height: 103px;
+  height: auto;
+  margin-bottom: 10px;
+  overflow: hidden;
+}
+
+.post-toprow-column2 {
+  position: relative;
+  min-width: 201px;
+  width: 201px;
+  overflow: hidden;
+
+  @include breakpoint(mobile) {
+    min-width: 94px;
+    width: 94px;
+    height: 103px;
+  }
+
+  @include breakpoint(phone) {
+    min-width: 94px;
+    width: 94px;
+    height: 103px;
+  }
```

Comment:
it looks like the stuff in `mobile` and `phone` is the same - the `mobile` breakpoint should make the `phone` breakpoint redundant here (the `mobile` one is `max-width: 800px` and the `phone` is `max-width: 580px`, so `mobile` will cover all the screen widths that `phone` will)

Reply:
got rid of the `phone` breakpoint